### ISSUE TITLE
In Beta Testing: (User feedback) Provide a clear way for screen reader to close app menu

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#93]
Log a bug against MAUI, as some method (perhaps a Close button) should be accessible to the screen reader by default.

In the meantime, consider adding a "Close menu" item at the bottom of the list of items in the menu.

**Comment by gbarkerz** [comment]
The app menu now contains a "Close menu" item.
